README.md

```diff
@@ -49,6 +49,20 @@ this may prompt you for sudo access.
     grunt install
     grunt uninstall
 
+## Building a RPM package
+
+We use Docker to build the RPM package in a clean environment, this also avoids
+to polute your environment.
+
+To build a RPM package run the following command at the root of the git repository:
+
+    docker run --rm -it \
```

Comment:
@colinbdclark done! :
`grunt buildrpm` builds a package in a Fedora environment
`grunt buildRpmDocker` builds a package using a Fedora Docker container